Apromore-Frontend/src/bpmneditor/scripts/editor.js

```diff
@@ -1,542 +0,0 @@
-            canvas.viewbox(false); // trigger recalculate the viewbox
-            canvas.zoom('fit-viewport', 'auto'); // zoom to fit full viewport
-        }
-    }
-
-    zoomIn() {
-        // this.actualEditor.get('editorActions').trigger('stepZoom', { value: 1 });
-        if (this.actualEditor) {
-            var canvas = this.actualEditor.get('canvas');
-            canvas.zoom(canvas.zoom() * 1.1);
-        }
-    }
-
-    zoomOut() {
-        // this.actualEditor.get('editorActions').trigger('stepZoom', { value: -1 });
-        if (this.actualEditor) {
-            var canvas = this.actualEditor.get('canvas');
-            canvas.zoom(canvas.zoom() * 0.9);
-        }
-    }
-
-    zoomDefault() {
-        // editorActions.trigger('zoom', { value: 1 });
-        if (this.actualEditor) {
-            var canvas = this.actualEditor.get('canvas');
-            canvas.zoom(1);
-        }
-    }
-
-    createShape(type, x, y, w, h) {
-        var modelling = this.actualEditor.get('modeling');
-        var parent = this.actualEditor.get('canvas').getRootElement();
-        //console.log('parent', parent);
-        var shape = modelling.createShape({type:type, width:w, height:h}, {x:x, y:y}, parent);
-        return shape.id;
-    }
-
-    updateProperties(elementId, properties) {
-        var modelling = this.actualEditor.get('modeling');
-        var registry = this.actualEditor.get('elementRegistry');
-        modelling.updateProperties(registry.get(elementId), properties);
-    }
-
-
-    createSequenceFlow (source, target, attrs) {
-        var attrs2 = {};
-        Object.assign(attrs2,{type:'bpmn:SequenceFlow'});
-        if (attrs.waypoints) {
-            Object.assign(attrs2,{waypoints: attrs.waypoints});
-        }
-        var modelling = this.actualEditor.get('modeling');
-        var registry = this.actualEditor.get('elementRegistry');
-        var flow = modelling.connect(registry.get(source), registry.get(target), attrs2);
-        //console.log(flow);
-        return flow.id;
-    }
-
-    createAssociation(source, target, attrs) {
-        var attrs2 = {};
-        Object.assign(attrs2,{type:'bpmn:Association'});
-        if (attrs.waypoints) {
-            Object.assign(attrs2,{waypoints: attrs.waypoints});
-        }
-        var modelling = this.actualEditor.get('modeling');
-        var registry = this.actualEditor.get('elementRegistry');
-        var assoc = Object.assign(assoc, modelling.connect(registry.get(source), registry.get(target), attrs2));
-        return assoc.id;
-    }
-
-    highlight(elementId) {
-        //console.log("Highlighting elementId: " + elementId);
-        var self = this;
-        var element = self.actualEditor.get('elementRegistry').get(elementId);
-        var modelling = self.actualEditor.get('modeling');
-        //console.log(element);
-        modelling.setColor([element],{stroke:'red'});
-    }
-
-    colorElements(elementIds, color) {
```

Comment:
As discussed, some of the removed functions (which are currently not used), can be used for the conformance checking front end (e.g. colorElements, createShape, createSequenceFlow). We can add them back in or replace them with more relevant functions instead of removing them completely.